__tests__/apiRequests.test.js

```diff
@@ -210,21 +210,66 @@ describe('APIs', () => {
         .get('/api/reviews')
         .expect(200)
         .then(({ body }) => {
-          const {reviewsArr }= body;
+          const { reviewsArr } = body;
           expect(reviewsArr).toHaveLength(13);
-          expect(reviewsArr).toBeSorted({descending: true})
-            expect(reviewsArr[4]).toEqual(
-              expect.objectContaining({
-                review_id: 3,
-                title: 'Ultimate Werewolf',
-                category: 'social deduction',
-                owner: 'bainesface',
-                review_img_url: 'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
-                created_at: '2021-01-18T10:01:41.251Z',
-                votes: 5,
-                comment_count: 3,
-              })
-            );
+          expect(reviewsArr).toBeSorted({ descending: true });
+          expect(reviewsArr[4]).toEqual(
+            expect.objectContaining({
+              review_id: 3,
+              title: 'Ultimate Werewolf',
+              category: 'social deduction',
+              owner: 'bainesface',
+              review_img_url:
+                'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+              created_at: '2021-01-18T10:01:41.251Z',
+              votes: 5,
+              comment_count: 3,
+            })
+          );
+        });
+    });
+  });
+  describe.only('9. /api/reviews/:review_id/comments responds with', () => {
+    test('status 200 OK , returns an array of comments of the given review_id', () => {
+      return request(app)
+        .get('/api/reviews/3/comments')
+        .expect(200)
+        .then(({ body: { commentsArray } }) => {
+          expect(commentsArray.length).toBe(3);
+          expect(commentsArray[0]).toEqual(
```

Comment:
Rather than check a single one, i'd like you to do this as a loop to check them all